evm-adapters/src/lib.rs

```diff
@@ -179,16 +179,46 @@ pub trait Evm<State> {
 #[cfg(test)]
 mod test_helpers {
     use super::*;
-    use dapp_solc::SolcBuilder;
-    use ethers::{prelude::Lazy, utils::CompiledContract};
+    use ethers::{
+        prelude::Lazy,
+        solc::{
+            artifacts::CompactContract, ArtifactOutput, Project, ProjectCompileOutput,
+            ProjectPathsConfig,
+        },
+    };
     use std::collections::HashMap;
 
-    pub static COMPILED: Lazy<HashMap<String, CompiledContract>> =
-        Lazy::new(|| SolcBuilder::new("./testdata/*.sol", &[], &[]).unwrap().build_all().unwrap());
+    pub static COMPILED: Lazy<HashMap<String, CompactContract>> = Lazy::new(|| {
+        // NB: should we add a test-helper function that makes creating these
+        // ephemeral projects easier?
+        let paths =
+            ProjectPathsConfig::builder().root("testdata").sources("testdata").build().unwrap();
+        let project = Project::builder()
+            .paths(paths)
+            .ephemeral()
+            .artifacts(ArtifactOutput::Nothing)
+            .build()
+            .unwrap();
+        let output = project.compile().unwrap();
+        let compiled = match output {
+            ProjectCompileOutput::Compiled((out, _)) => out,
+            _ => panic!(),
+        };
```

Comment:
This fails to compile `Cheatcodes.sol` because of a:

```
  "/Users/gakonst/oss/dapptools-rs/evm-adapters/testdata/CheatCodes.sol:6:1: ParserError: Source \"/Users/gakonst/oss/dapptools-rs/evm-adapters/testdata/DsTest.sol\" not found: File outside of allowed directories.\nimport \"./DsTest.sol\";\n^--------------------^\n",
```

error

I do not get it, why are we getting this error?